x-pack/plugins/search_inference_endpoints/kibana.jsonc

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "plugin",
+  "id": "@kbn/search-inference-endpoints",
+  "owner": "@elastic/enterprise-search-frontend",
```

Comment:
We'll want to update this to the new team now: `@elastic/search-kibana`

Reply:
Updated